src/flow-util.c

```diff
@@ -249,6 +249,8 @@ void FlowEndCountersRegister(ThreadVars *t, FlowEndCounters *fec)
             name = "flow.end.state.established";
         } else if (i == FLOW_STATE_CLOSED) {
             name = "flow.end.state.closed";
+        } else if (i == FLOW_STATE_DROPPED) {
+            name = "flow.end.state.dropped";
```

Comment:
Even if we don't have the end flag for this state, we need this, to prevent having an id zero when adding final stats for the flow - from what I could understand.

Reply:
with regards to the flow state eve field, I'm thinking that it's useful info to be able to know if a dropped flow reached an "established" state, for instance. So... I need to put some thought into this, I guess.